tmp/addons/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
@eviljeff  i'm guessing those files don't affect the merge right ? They're marked to be ignored in `.gitignore`